WFInfo/Data.cs

```diff
@@ -1159,7 +1157,7 @@ public void SetJWT(HttpResponseHeaders headers)
         {
             foreach (var item in headers)
             {
-                if (!item.Key.Contains("Authorization")) continue;
+                if (!item.Key.Contains("authorization")) continue;
```

Comment:
Weird, is it intended?

Reply:
Not sure, don't know when it changed either